grype/matcher/apk/matcher.go

```diff
@@ -135,3 +135,81 @@ func vulnerabilitiesByID(vulns []vulnerability.Vulnerability) map[string][]vulne
 
 	return results
 }
+
+func (m *Matcher) findApkPackage(store vulnerability.Provider, d *distro.Distro, p pkg.Package) ([]match.Match, error) {
```

Comment:
Praise: It's awesome to break this out into a distinct function! 🎉

Reply:
Thanks!